src/vm/arm/crthelpers.S

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+//
+
+// ==++==
+//
+
+//
+// ==--==
+// ***********************************************************************
+// File: crthelpers.S
+//
+// ***********************************************************************
+
+#include "unixasmmacros.inc"
+#include "asmconstants.h"
+
+// JIT_MemSet/JIT_MemCpy
+//
+// It is IMPORANT that the exception handling code is able to find these guys
+// on the stack, but to keep them from being tailcalled by VC++ we need to turn
+// off optimization and it ends up being a wasteful implementation.
+//
+// Hence these assembly helpers.
+// 
+//EXTERN_C void __stdcall JIT_MemSet(void* _dest, int c, size_t count)
+LEAF_ENTRY JIT_MemSet, _TEXT
+
+        cmp r0, #0
+        bxeq lr
+        
```

Comment:
Please add an attempt to access the [r0] like we do for amd64. This ensures that if the parameter is wrong, we fail here and not inside of the memset, which is important due to the fact that we handle hardware exceptions only in the coreclr code.